pkgs/development/libraries/sharness/default.nix

```diff
@@ -41,11 +41,13 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  passthru.SHARNESS_TEST_SRCDIR = finalAttrs.finalPackage + "/share/sharness";
```

Comment:
Could we not use $out here?

Reply:
First, being a passthru and not being a shell variable, I would suspect that $out is just wrong here. Second, this variable must point to the resulting store path after all overrides have been applied. Overriding sharness with augmented test functions belongs to the package's API.

Reply:
Right, I think we also can't use placeholder 🤔